app/static/app/js/components/ErrorMessage.jsx

```diff
@@ -11,22 +11,42 @@ class ErrorMessage extends React.Component {
 
     constructor(props){
         super(props);
+        this.state = { hidden: false }
         this.close = this.close.bind(this);
     }
 
     close(){
         const [parent, prop] = this.props.bind;
-        parent.setState({[prop]: ""});
```

Comment:
Changing the logic here is going to create issues in other React components that use `ErrorMessage`.